iss-79 issue の追加時にGitHub へIssue を追加する

koala777 opened the pull request:
・TimeCard からIssue 追加時にGitHub へIssue を追加する
・issue Noをクリックすることで、GitHub の対象のissue に遷移する

koala777 committed:
iss-79 issue の追加時にGitHub へIssue を追加する

・TimeCard からIssue 追加時にGitHub へIssue を追加する
・issue Noをクリックすることで、GitHub の対象のissue に遷移する

causeway commented:
Issue 登録のView に GitHub に登録するかのフラグが必要になります。
このチェックがON の場合に、GitHub に登録する仕組みです。
(※Project が GitHub 連携を行っている場合のみ GitHub に登録するフラグが表示され、それ他はこのチェックボックスは非表示になります。)
現状だとすべてのIssue が GitHub に登録されてしまうと思います。

koala777 committed:
iss-79 PR指摘反映

・Project が GitHub 連携されているときは、New Issue に GitHub チェックボックスを表示する
・New Issue の GitHub がチェックされているときは GitHub へ Issue を登録
・GitHub へ登録されていない Timecard の Issue 一覧で、GitHub へのリンクは表示しない
